src/PrettyPrinterTrait.php

```diff
@@ -54,7 +58,9 @@ public function endTest(Test $test, float $time): void
         $name = preg_replace('/^test /', '', $name, 1);
 
         // Get the data set name
-        $name = $this->handleDataSetName($name, $testMethodName[1]);
+        if ($dataSet) {
+            $name .= ' [ ' . preg_replace('/^"|"$/', '', $dataSet) . ' ]';
```

Comment:
Note: I used this instead of `trim($dataset, '"')` because the dataset may end with a quote (double quotes) and that would remove both from the end. This will only match a single quote from the beginning and end.